specification/support/resource-manager/readme.md

```diff
@@ -26,18 +26,44 @@ These are the global settings for the Support API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-04
+tag: package-preview-2022-09
 ```
 
 
+### Tag: package-preview-2022-09
+
+These settings apply only when `--tag=package-preview-2022-09` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2022-09'
+input-file:
+  - Microsoft.Support/preview/2022-09-01-preview/support.json
+suppressions:
+  - code: RPC-Put-V1-01
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The path for put operation must be under a subscription and resource group. Justification: Suppressing this rule since path for support ticket resource doesn't contain resource group"
+  - code: RPC-Uri-V1-05
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The resource name parameter supportTicketName should be defined with a pattern restriction. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Put-V1-11
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: Any Put MUST contain 200 and 201 return codes. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Patch-V1-03
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The property {propertyName} in the request body either not apppear in the resource model or has the wrong level. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Get-V1-05
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
```

Comment:
If needed, apply the suppressions at particular definition, or code path level, not for whole file. Otherwise that will suppress all the future wrong usages as well.

Reply:
These suppressions are used for more than one definition and since we follow same pattern across, it made sense to have it for the whole file. So, keeping it as is.

Reply:
Thank you. But for the below points, it can be considered:

1. RPC-Uri-V1-05: Why can't this be fixed for new properties being introduced, add pattern to new properties and apply linter suppressions only for existing properties if needed.

2. RPC-Patch-V1-03: For any new Patch APIs, this can be fixed, essentially should be fixed. So, apply suppression only for specific existing ones.

3. RPC-Get-V1-05: This also seems specific to FileWorkspaceDetails, so apply suppression only for this one.

Recommendation is to improve any new additions to the swagger and avoid previously non-recommended behavior in order to keep parity.

Reply:
> Thank you. But for the below points, it can be considered:
> 
> 1. RPC-Uri-V1-05: Why can't this be fixed for new properties being introduced, add pattern to new properties and apply linter suppressions only for existing properties if needed.
> 2. RPC-Patch-V1-03: For any new Patch APIs, this can be fixed, essentially should be fixed. So, apply suppression only for specific existing ones.
> 3. RPC-Get-V1-05: This also seems specific to FileWorkspaceDetails, so apply suppression only for this one.
> 
> Recommendation is to improve any new additions to the swagger and avoid previously non-recommended behavior in order to keep parity.

Please respond to this, rest of the PR looks fine. Just these suppressions need to be defined correctly.

Reply:
Applied the changes to this suppressions specific to code path wherever applicable

Reply:
RPC-Get-V1-05 , Can you please explain the justification here ? Simply stating that we cannot support is not enough , what is the reason ?


For all the other ones , please suppress for specific lines as called out by vikas. The older implementations will not get flagged and we want to only suppress for specific instances that you are creating newly. 

If you have already fixed some of the suppressions then remove it from the readme file.

Reply:
please feel free to send me an email if you want to discuss this directly also.

Reply:
I have removed couple of suppression rules(RPC-Uri-V1-05 and RPC-Patch-V1-03), since we have taken care of those for new APIs. And for RPC-Get-V1-05, we explained in detail to the reviewer during private preview PR([[Hub Generated] Review request for Microsoft.Support to add version preview/2022-09-01-preview by raghumunukutla · Pull Request #8647 · Azure/azure-rest-api-specs-pr (github.com)](https://github.com/Azure/azure-rest-api-specs-pr/pull/8647)) and got a sign-off which is the reason why it was not mentioned in this PR.

I have emailed detailed explanation with screenshots from private preview PR and also the explanation.